packages/ui-drilldown/src/Drilldown/README.md

```diff
@@ -973,10 +1480,29 @@ guidelines: true
 ---
 <Guidelines>
   <Figure recommendation="yes" title="Do">
-    <Figure.Item>TODO: guidelines</Figure.Item>
+    <Figure.Item>
+      Use to display tree structures (instead of the TreeBrowser)
+    </Figure.Item>
+    <Figure.Item>
+      Use instead of flyout menus
+    </Figure.Item>
+    <Figure.Item>
+      Use for radio or checkbox type interactions
+    </Figure.Item>
+    <Figure.Item>
+      Make the text within Drilldown direct so users can quickly decide on an action
```

Comment:
hmm I dont understand this one, what does it mean?

Reply:
Changed it a bit